mongoCollections.ts

```diff
@@ -0,0 +1,18 @@
+import { dbConnection } from "./mongoConnection";
+
+// This will allow you to have one reference to each collection per app
+function getCollectionFn(collection: any) {
+	let _col: any = undefined;
+
+	return async () => {
+		if (!_col) {
+			const db = await dbConnection();
+			_col = await db.collection(collection);
+		}
+
+		return _col;
+	};
```

Comment:
if you could add a return type to the returned-function that would be good because that will be used in other places

Reply:
@aej11a I'm not sure what the return type would be here. I tried making it `Collection` or `Collection<Document>`, but it errors out when I try to do this. This one is a problem I haven't been able to resolve yet, unfortunately.